docs/troubleshooting-rbe.md

```diff
@@ -37,3 +37,36 @@ If you'd like to add Mac executors to your BuildBuddy Cloud account, please [con
 This error occurs when your build is configured for darwin (Mac OSX) CPUs, but attempting to run on Linux executors. Mac executors are not included in BuildBuddy Cloud's free-tier offering.
 
 If you'd like to add Mac executors to your BuildBuddy Cloud account, please [contact our sales team](/request-demo/).
+
+## WARNING: Remote Cache: UNAVAILABLE: io exception
+
+This error may occur when Bazel fails to properly maintain a long-running TCP connection to BuildBuddy.
+
+To check whether this is the case, try running Bazel with `--remote_grpc_log=grpc.log` to capture the gRPC traffic
+between Bazel and BuildBuddy. The log file will be in protobuf format. To convert it to JSON format, download the [BuildBuddy CLI](/docs/cli) and run `bb print --grpc_log=<path-to-file>/grpc.log`.
+
+In the log, you may see network errors such as the following:
+
+```json
+  "status":  {
+    "code":  14,
+    "message":  "io.netty.channel.unix.Errors$NativeIoException: readAddress(..) failed: Connection reset by peer"
+  },
+```
+
+This typically happens when there is a proxy or gateway (e.g. AWS NAT Gateway) in between Bazel and BuildBuddy that is terminating idle connections too quickly.
+
+When this happens, try the following Linux network settings:
+
+```bash
+# Lowered from default value: 7200
+sudo sysctl -w net.ipv4.tcp_keepalive_time=180
+# Lowered from default value: 75
+sudo sysctl -w net.ipv4.tcp_keepalive_intvl=60
+# Lowered from default value: 9
+sudo sysctl -w net.ipv4.tcp_keepalive_probes=2
```

Comment:
I would raise this to 4 or 5. My math was a little handwavy earlier :-P 
Once keepalive is triggered, the initial time (180) is not used anymore. When keepalive first comes into effect, the time is 180+60\*2, but on subsequent checks the timeout becomes only 60*2 before connection is considered dead.

Reply:
bump the number to 5. 👍